Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Base image to use, this must be set as the first line
+FROM ubuntu
+
+LABEL maintainer="fyuxin@uwaterloo.ca"
+
+# Commands to update the image
+RUN apt-get update && apt-get install -y \
+    wget \
+    unzip \
+    libfontconfig \
+    libgconf2-4 \
+    libglib2.0-0 \
+    libnss3 \
+    libx11-6 \
+    build-essential \
+    python-dev \
+    python-setuptools \
+    python-pip \
+    python-smbus \
+    libncursesw5-dev \
+    libgdbm-dev \
+    libc6-dev \
+    zlib1g-dev \
+    libsqlite3-dev \
+    libssl-dev \
+    openssl \
+    libffi-dev
+
+WORKDIR /usr/local/share/
```

Comment:
Why we need to download chromedriver by hand?

Check [here](https://github.com/xingweitian/fund-my-watcard/blob/master/fund_my_watcard/watcard#L71), I use `webdriver_manager` to download the corresponding driver for Chrome.

So in my opinion, we actually don't need to download a specific chromedriver.

Also, I didn't see where are the commands to install Chrome, have you tried the docker image? This version of the dockerfile maybe cannot work well. [Here](https://www.itzgeek.com/how-tos/linux/ubuntu-how-tos/install-google-chrome-on-ubuntu-16-04.html) should be useful for you.

Reply:
In addition, let's think about if it is better to install fund-my-watcard from source code, or from pip.